### ISSUE TITLE
Font-weight thing

### ISSUE THREAD

**Issue by kissane** [OpenNews/opennews-source#139]
Yan0 notes:

> looks like you need to update the site CSS so that 'strong' uses 'font-family: freight-text-pro-bold;' rather than 'freight-text-pro'

Which maybe gets to that unreproduceable font thing Lindsay was seeing awhile back with missing bolding.

**Comment by beep** [comment]
`[ a wild BEEP appears ]`

> looks like you need to update the site CSS so that 'strong' uses 'font-family: freight-text-pro-bold;' rather than 'freight-text-pro'

Oh, hm. I don’t think that’s correct? In Typekit-ese, the family should be `freight-text-pro`, regardless of weight. (Which would get set with a `font-weight` value in our CSS.)

(My hunch on #136 was that it was down to poor contrast on my part; darkening the color might help?)

`[ BEEP used HUNCH. It’s not very effective… ]`

**Comment by kissane** [comment]
THE SUMMONING TOGGLE GOT LEFT ON AGAIN

*thx beep we will investigaaaaate*

**Comment by kissane** [comment]
Soooo I'm not sure what to do here. The bold looks legible to me, but there's apparently a situation in which it doesn't show up for some people at all. I'm inclined to leave this open and try to get screenshots and details the next time it arises.

**Comment by ryanpitts** [comment]
was David Yanofsky one of the people seeing this issue? maybe still screenshottable by him?

**Comment by beep** [comment]
If y’all can’t reproduce it internally, I might suggest looking into darkening the color a bit. Here’s what `strong` looks like now on my machine:

<img width="374" alt="screen shot 2017-04-03 at 11 12 31 am" src="https://cloud.githubusercontent.com/assets/124493/24623793/874c7872-185e-11e7-8ac4-e590ee2ee9c8.png">

Here’s what it looks like a ~half a shade darker, at `#333`:

<img width="353" alt="screen shot 2017-04-03 at 11 12 49 am" src="https://cloud.githubusercontent.com/assets/124493/24623800/8b793688-185e-11e7-936b-2cbda94dda33.png">

Miiiiiight give it an added visibility boost?

(apologies for sealioning in here, hi sorry showin’ myself out)

**Comment by kissane** [comment]
Thanks, @beep! This is awesome to have in a pocket!

@ryanpitts, I've asked David and will report back if he can replicate.

**Comment by ryanpitts** [comment]
@beep appreciate it! One thing I'm wondering about is how safe it might be to assign a color directly to `b, strong` ... just in case something else that has a color to it ends up wrapping one of those tags. But also wary of changing the `src-text-default` color as well.

**Comment by beep** [comment]
@ryanpitts _Super_ great question, thank you! I think you’re right to be leery of changing `src-text-default`. (Unless, of course, you’d like to darken the text color globally, which, hey, that is also an option!)

As for adding a style to `strong` directly, I think that’d be 100% fine. I think you have two options in this area:

1. Adding a global `strong` rule to `_base.scss`. I think this is fine, since we’re not really leaning on `strong` in other parts of the design—at least, not that I remember—and a global style’s probably a-OK.
2. But! If that feels weird to you, it’d be tooootally fine to scope the change to `.article-main`. (i.e., `.article-main strong { color: #333; }`) If I was doing that, I’d add that to `_core.scss`.

Does that help at all? Basically, I think either approach is fine, it’s really about what feels right to you.

**Comment by ryanpitts** [comment]
@beep yep, that helps a ton. Will wait to see if @kissane hears back with some screenshots, and then deploy whatever makes most sense based on that.

**Comment by kissane** [comment]
This is what we got back from David, who is on Chrome Version 56.0.2924.87

![screen shot 2017-04-03 at 11 40 28 am](https://cloud.githubusercontent.com/assets/1910034/24627686/c492025e-1883-11e7-8681-2e14b10b3ca8.png)

**Comment by kissane** [comment]
(The org titles are meant to be bold, there)

**Comment by beep** [comment]
@kissane Thank you! Could I ask if you have a URL for that page?

**Comment by kissane** [comment]
Yes! https://source.opennews.org/articles/same-diff-trumps-budget-proposal/

Yes that would help, urn.

**Comment by beep** [comment]
HM. So yeah, this seems like a Chrome 56 bug. (Or some combination between Chrome 56, macOS, and Typekit.)

Here’s what I see on Chrome 57:

<img width="628" alt="screen shot 2017-04-03 at 1 35 29 pm" src="https://cloud.githubusercontent.com/assets/124493/24630890/320ee7c2-1873-11e7-9f16-9fdbf3c34014.png">

Here’s what I see on Chrome 56 (via Browserstack):

<img width="630" alt="screen shot 2017-04-03 at 1 37 10 pm" src="https://cloud.githubusercontent.com/assets/124493/24630916/4eaeee4a-1873-11e7-82cb-f444faa578cd.png">

The weights are definitely not applying correctly, and I can’t seem to “fix” that with CSS. Definitely seems like a browser rendering bug.

(Worth noting, however, that [darkening the color works just fine](https://cloud.githubusercontent.com/assets/124493/24631021/bd468d90-1873-11e7-8bda-8198c85a3bac.png). The weight’s still busted, however.)

Given that Chrome’s one o’ them evergreen browsers, I miiiiiight suggest leaving it as-is, and let browser upgrades sort this out. Unless, of course, this is affecting a bunch of users! In which case, ignore me!

**Comment by beep** [comment]
(On re-previewing the images, it seems like the weights _might_ be applied, but the artifacting on Browserstack might be obscuring it. So maybe it’s not showing the bug after all? idk idk)

**Comment by kissane** [comment]
I think we note this one as a Known Thing—thank you SO MUCH for the help, @beep!

**Comment by beep** [comment]
Any time, @kissane! Thanks for letting me sealion in here. <3
